core/src/test/resources/com/google/googlejavaformat/java/testdata/B20128760.output

```diff
@@ -89,19 +89,13 @@ class B20128760 {
   {
     Stream<ItemKey> itemIdsStream =
         stream(members)
-            .flatMap(
-                m ->
-                    m.getFieldValues()
+            .flatMap(m -> m.getFieldValues()
                         .entrySet()
```

Comment:
I agree with Jake. IMO, `entrySet()` and all the lines afterwards should be indented +4, rather than, say, +12, as is the case currently.